README.md

```diff
@@ -86,14 +86,15 @@ julia> path = fetch_ba_group("ladybug")
 You can directly construct a nonlinear least-squares model based on [NLPModels](http://juliasmoothoptimizers.github.io/NLPModels.jl/latest/):
 
 ```julia
-julia> model = BundleAdjustmentModel("problem-49-7776-pre", "ladybug")
+julia> model = BundleAdjustmentModel("problem-49-7776-pre")
 BundleAdjustmentModel{Float64, Vector{Float64}}
 ```
 
-You can also construct a nonlinear least-squares model by giving the constructor the path to the archive:
+You can also construct a nonlinear least-squares model by giving the constructor the path to the archive.
+You must also set `direct_path = true`
```

Comment:
What does "direct path" mean?

Reply:
"direct_path" means you directly give the path to a problem file instead of using the system of artifacts.
It should almost never be the case but I left it in case someone wants to modify some of the problems themselves.

Reply:
I don't think we want to encourage that.

Reply:
I will delete it in this case.